src/uwtools/resources/jsonschema/global-equiv-resol.jsonschema

```diff
@@ -0,0 +1,25 @@
+{
+  "properties": {
+    "global_equiv_resol": {
+      "additionalProperties": false,
+      "properties": {
+        "execution": {
+          "$ref": "urn:uwtools:execution"
+        },
```

Comment:
Should we consider defining an `execution_serial` variant here that rejects inapplicable item(s) like `mpicmd`? It might be better to communicate our expectations to the user at config-validation time rather than just ignoring accepted entries. If so, we might use that new serial schema for `esg_grid`, too. Are there others?

Reply:
I think the YAML block would still be called just `execution`, but it'd be validated by a schema with simplified content, without `mpiargs` or `mpicmds`. Not sure about `threads` -- maybe also not?

Reply:
Added this in my open PR #467

Reply:
We have the `execution-serial` schema now, so I'll resolve this comment and add a suggestion on the appropriate line.